### PR TITLE
Feature/claudia 5210 travis configuration

### PR DESCRIPTION
**Pull request by flopezag**
#### Reviewers

@chemaper, @jframos  
#### Description
- Configure Travis for the glancesync component.
- Add badge from develop to the README.rst file
#### Testing

Testes in Travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - "2.7"
+
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+  - pip install -r requirements_dev.txt
+
+#before_script:
+#  - sudo touch /var/log/fiware-skuld
+#  - sudo chown travis /var/log/fiware-skuld
```
Comment (jframos) [inline .travis.yml]: Do we need these lines?

Comment (flopezag) [inline .travis.yml]: nop

**Comment by jframos** [comment]
LGTM

**Comment by chemaper** [comment]
LGTM

**Pull request merged**